server/core/storelimit/store_limit.go

```diff
@@ -25,29 +25,56 @@ const (
 	Unlimited = float64(100000000)
 )
 
+const influence = 1000
```

Comment:
How about moving it below line 25?

Reply:
yes, it will not be nil even if the rate per second is zero, I will remove this condition.